### ISSUE TITLE
New Feature for find-mentor (Interview Wizard)

### ISSUE THREAD

**Issue by emincanoguz11** [findmentor-network/find-mentor#276]
Hello everyone,

I have a idea for find-mentor. The aim of feature is the user(developer) find what is user's career goal to easiest way and the system is prepared user to interview questions. In my opinion, this attribute is very important for career goals. Because lots of sites have interview questions or finding the way but I don't see any all-in-one program for this feature. This feature motivation is like that.

You can check basic diagram for this feature in figure.

![findmentor_2](https://user-images.githubusercontent.com/47830986/106896062-1ed80080-6702-11eb-95fa-e51b03833db9.png)

What do you think about this feature? I am waiting reviews.

Sincerely,
Emin Can

**Comment by cagataycali** [comment]
Can we collect interview questions and list in somewhere firstly?

**Comment by emincanoguz11** [comment]
Yes we collect and we create a wizard for users' interview answering.
